components/network/quagga/quagga.p5m

```diff
@@ -0,0 +1,132 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+# Copyright (c) 2011, 2015, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2020 Olaf Bohlen <olbohlen@eenfach.de>
+#
+
+<transform file path=usr.*/man/.+ -> default mangler.man.stability volatile>
+set name=pkg.fmri \
+    value=pkg:/system/network/routing/quagga@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="Quagga - Quagga Routing Software"
+set name=pkg.description \
+    value="Quagga is a routing software suite, providing implementations of OSPFv2, OSPFv3, RIP v1 and v2, RIPng, ISIS and BGP-4"
+set name=com.oracle.info.description value="the Quagga routing software"
+set name=com.oracle.info.tpno value=$(TPNO)
+set name=info.classification \
+    value="org.opensolaris.category.2008:System/Administration and Configuration"
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+#set name=org.opensolaris.arc-caseid value=PSARC/2005/571
+#set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+file OpenIndiana/README.OpenIndiana path=etc/quagga/README.OpenIndiana
+file path=etc/quagga/bgpd.conf.sample
```

Comment:
Can we deliver some actual files, not *sample, which would work by default?

Reply:
Does it make sense to move *sample files somewhere to usr/sharer/doc/quagga/examples?

Reply:
Unfortunately "default" configs will not make any sense to quagga, so no.
The path can be discussed, but we also ship sample confs for apache in /etc/httpd/2.X/conf.d.sample ...so I do think this is a valid location

Reply:
We have etc/apache2/2.4/samples-conf/file no etc/apache2/2.4/conf.d/file.sample, I suppose it's better to move sample files out of /etc at least to simplify auto-completion, when actual files are in place.